source/_components/tts.markdown

```diff
@@ -22,6 +22,10 @@ tts:
   - platform: google
 ```
 
+<p class='note'>
+If you are running Home Assistant over SSL or from within a container, you will have to setup a base URL (`base_url`) inside the [http component](/components/http/).
```

Comment:
If a reverse proxy is in use, `base_url` isn't needed. I'm not sure though how to call that out, without adding confusion though.

Reply:
I agree with @DubhAd. You could point to it without saying why.
Like:

```text
Depending on your setup, you might need to set a base URL (`base_url`) inside the [http component](/components/http/).
```

Reply:
With this change I had just moved what was currently in place as I thought that was already the correct wording :smile:  I can change it to your suggestion but I'm not sure how to add more commits to an open pull request or if I should just create a new pull request. Any advice would be helpful :+1: Cheers

Reply:
Simply choose the "Files changed" tab here on GitHub (top of the PR), hit the edit button (pencil) button on the file header, and update 👍